.vuepress/get_sidebar.js

```diff
@@ -101,64 +100,68 @@ module.exports = {
                           //The parent of this is a new child group. So we need to create it as such and make it a parent.
                           let new_parent = {};
                           new_parent.title = last_item.title;
-                          new_parent.path = last_item.path;
+                          // new_parent.path = last_item.path;
```

Comment:
@bys1123 FYI, this is the relevant change to the sidebar. To revert you uncomment this line and comment the one below it so code looks like this. 
```
 new_parent.title = last_item.title;
new_parent.path = last_item.path;
```
Then you remove the three new lines below:
```
//Add first child
let entry = new Array(last_item.path, 'Introduction'); //last_item.title+
new_parent.children.push(entry);
```